html-web-page/index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+  <h1>
+    Jelly of the Month Club
+  </h1>
+
+  <h2>
+    Make all your friends jelly of your stock of jelly!
+  </h2>
+  <p>
+    Here at JOTMC Inc. We are constantly striving to supply our
+    customers with the highest-quality and most unique jellies on a
+    monthly basis. For the jelly lover in your life(or yourself!),
+    this is the gift that keeps on giving all year round. We offer
+    three tiers of monthly jelly joy, and each one offers their own
+    perks. Here is a breakdown of each, as well as your monthly price:
+  </p>
+  <table>
+    <thead>
+      <tr>
+        <th scope="col">                </th>
+        <th scope="col">Basic Plan</th>
+        <th scope="col">Deluxe Plan</th>
+        <th scope="col">Jelly Giant Plan</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th scope="row">Three new curated jellies per month</th>
+        <td>&check;</td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <th scope="row">Monthly jellies come with commemorative mason jars</th>
+        <td>     </td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <th scope="row">Free bonus jar of preserves bi-monthly</th>
+        <td>     </td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <th scope="row">Free yearly tickets to Jelly Con</th>
+        <td>    </td>
+        <td>    </td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <th scope="row">Monthly Pricing</th>
+        <td>&#36 30</td>
+        <td>&#36 50</td>
+        <td>&#36 60</td>
+      </tr>
+    </tbody>
+  </table>
+  <p>
+    Like what you see? Become a part of the JOTMC family today!
+  </p>
+  <label for="name">Your Name:</label>
+
+  <input type="text" id="name" name="name" required minlength="4" maxlength="8" size="10">
+
+
+
+  <select name="pets" id="pet-select">
+    <option value="">Select Your Plan:</option>
+    <option value="">etc..</option>
+    <option value="">etc..</option>
+    <option value="">etc..</option>
+    <option value="">etc..</option>
+    <option value="">etc..</option>
+    <option value="">etc..</option>
+  </select>
+  <button class="favorite styled" type="button">
+    Submit
+  </button>
```

Comment:
When you have a submit button for a form you dont want to change it to `type="button"` since the default is `type="submit"` anyway.